db/migrate/20200831194208_rename_webhook_targets_to_webhooks.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class RenameWebhookTargetsToWebhooks < ActiveRecord::Migration[6.0]
+  def change
+    rename_table :webhook_targets, :webhooks
```

Comment:
I don't have a strong opinion, both work for me.

Reply:
Could `webhooks` mean the history of the actual events?
But I don't have a strong opinion either.

Reply:
Good point, do rename it then.

Reply:
So we have an agreement to rename.